chart/templates/gotk/rbac.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app.kubernetes.io/instance: {{.Release.Namespace}}
+    app.kubernetes.io/version: latest
+  name: crd-controller-{{.Release.Namespace}}
+  namespace: {{.Release.Namespace}}
+rules:
+- apiGroups:
+  - source.toolkit.fluxcd.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- apiGroups:
+  - kustomize.toolkit.fluxcd.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- apiGroups:
+  - helm.toolkit.fluxcd.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- apiGroups:
+  - notification.toolkit.fluxcd.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - configmaps/status
+  verbs:
+  - '*'
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/instance: {{.Release.Namespace}}
+    app.kubernetes.io/version: latest
+  name: crd-controller-{{.Release.Namespace}}
+  namespace: {{.Release.Namespace}}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: crd-controller-{{.Release.Namespace}}
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: {{.Release.Namespace}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/instance: {{.Release.Namespace}}
+    app.kubernetes.io/version: latest
+  name: cluster-reconciler-{{.Release.Namespace}}
```

Comment:
The use of namespace in the names is inherited from GOTK so I think I'll keep it.
GOTK team would tell you there should be only one GOTK per cluster, we can support more than one.
However the GOTK seems to be configured such that it will assume one installation per namespace so their namespace in name makes some sense